default/scripting/techs/construction/ORBITAL_CON.focs.txt

```diff
@@ -6,13 +6,25 @@ Tech
     researchcost = 40 * [[TECH_COST_MULTIPLIER]]
     researchturns = 4
     tags = [ "PEDIA_CONSTRUCTION_CATEGORY" ]
-    effectsgroups =
+    effectsgroups = [
         EffectsGroup
             scope = And [
                 Planet
                 OwnedBy empire = Source.Owner
             ]
             effects = SetMaxSupply value = Value + 1
+            
+        EffectsGroup    // outpost supply increases 1 per turn up to max
```

Comment:
Tech is not a starting unlock, so outposts still will not grow towards max supply until this tech is researched.

I've not noticed if supply meter growth supposed to be affected by combat. This effectsgroup has no check, but STANDARD_SUPPLY_GROWTH does.

Reply:
I don't think that's a problem, as outposts have 0 max supply before then tech anyway.

Reply:
Tiny/Small planets?